roles/tests-integration/tests/common/mod.rs

```diff
@@ -145,8 +145,11 @@ impl TemplateProvider {
         TemplateProvider { bitcoind }
     }
 
-    fn stop(&self) {
-        let _ = self.bitcoind.client.stop().unwrap();
+    fn stop(&mut self) {
+        match self.bitcoind.stop() {
+            Ok(_) => (),
+            Err(e) => eprintln!("Failed to stop bitcoind: {:?}", e),
```

Comment:
we should still `panic!` here in case of `Err`

the difference now is that we don't expect this `Err` case to be triggered anymore

but in case it does, we don't want to mask that with a `eprintln!`